models.js

```diff
@@ -0,0 +1,10 @@
+const { Pool } = require('pg');
+const db = require('./db/connection');
+const format = require('pg-format');
+
+exports.getAllTopics = async () => {
+    const topics = await db.query(`
```

Comment:
rather than call this topics I would be more tempted to have this be the db response and restructure the rows from it, it read a little confusing to see topics.rows being returned